src/backend/mailbox_manager/tests/api/mail_domain/test_api_mail_domains_create.py

```diff
@@ -56,13 +61,26 @@ def test_api_mail_domains__create_authenticated():
 
     client = APIClient()
     client.force_login(user)
-    response = client.post(
-        "/api/v1.0/mail-domains/",
-        {
-            "name": "mydomain.com",
-        },
-        format="json",
-    )
+
+    domain_name = "test.domain.fr"
+
+    with responses.RequestsMock() as rsps:
```

Comment:
Is there a reason, not to use the responses "classic" way? By classic I mean using the test function decorator `responses.activate` along with 

``` python
    responses.add(
        responses.POST,
        url=re.compile(r".*/domains/"),
        json={
            "name": domain_name,
        },
        status=status.HTTP_201_CREATED,
    )
```

which seems (to me) a bit more readable.

Reply:
I haven't had the time to make this PR but I absolutely want to make a fixture to encapsulate these responses ! Do you have a working example of how `responses.activate` functions ?